adafruit_imageload/png.py

```diff
@@ -86,6 +87,13 @@ def load(
                 pal = palette(pal_size)
                 for i in range(pal_size):
                     pal[i] = file.read(3)
+        elif chunk == b"tRNS":
+            if size > len(pal):
+                raise ValueError("More transparency entries than palette entries")
+            trns_data = file.read(size)
+            for i in range(len(trns_data)):
+                if trns_data[i] == 0:
+                    pal.make_transparent(i)
```

Comment:
Can we `del trns_data` once we are done with it?

Reply:
Sure can! Latest commit does so now.